src/controllers/DappsStakingV3Controller.ts

```diff
@@ -188,5 +188,63 @@ export class DappsStakingV3Controller extends ControllerBase implements IControl
                 }
             },
         );
+
+        app.route('/api/v3/:network/dapps-staking/rewards/:period').get(async (req: Request, res: Response) => {
```

Comment:
It would be good to be able to filter rewards by type. I don't see scenario asking for all rewards.

Reply:
Yes I will add the filter here.